### PR TITLE
Modified functionality and colors to work accordingly

### PR DESCRIPTION
**Pull request by NathanDVT**

### CODE REVIEW

File: NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift [hunk 1]
```diff
@@ -4,6 +4,7 @@ import AVKit
 
 @available(iOS 13.0, *)
 public class DashboardViewModel: DashboardViewModelProtocol {
+
```
Comment (NathanDVT) [inline NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift:7]: Codacy Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5376515)

**Comment by NathanDVT** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5376515&bid=17629170)

File: NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift [hunk 2]
```diff
@@ -97,7 +98,7 @@ public class DashboardViewModel: DashboardViewModelProtocol {
     }
 
     public func getTrending() {
-        self.repo?.getTrending() { [weak self] result in
+        let x = self.repo?.getTrending() { [weak self] result in
```
Comment (NathanDVT) [inline NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift:101]: Codacy Issue found: [Variable name should be between 3 and 40 characters long: 'x'](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5376515)

Comment (NathanDVT) [inline NLibrary/Classes/DashboardHomeMVVM/DashboardViewModel.swift:101]: Codacy Issue found: [When using trailing closures, empty parentheses should be avoided after the method call.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5376515)

**Review by AkuaQ — APPROVED** [review]

**Pull request merged**